AirCasting/SessionViews/ChartView.swift

```diff
@@ -70,19 +58,72 @@ class UI_PollutionChart: UIView {
 
 struct ChartView: UIViewRepresentable {
     
+    @ObservedObject var stream: MeasurementStreamEntity
+    var thresholds: SensorThreshold
+    
     typealias UIViewType = UI_PollutionChart
     
     func makeUIView(context: Context) -> UI_PollutionChart {
         UI_PollutionChart()
     }
     
     func updateUIView(_ uiView: UI_PollutionChart, context: Context) {
+        let chartCreator = ChartEntriesCreator(stream: stream)
+        var entriesForDrawing = chartCreator.generateEntries()
+        entriesForDrawing.sort { (e1, e2) -> Bool in
+            e1.x < e2.x
+        }
+        
+        let dataSet = LineChartDataSet(entries: entriesForDrawing)
+        let data = LineChartData(dataSet: dataSet)
+        uiView.lineChartView.data = data
+        
+        //format data labels
+        formatData(data: data)
+        
+        //format line and dots
+        formatDataSet(dataSet: dataSet)
+    }
+    
+    func formatData(data: LineChartData) {
+        let formatter = NumberFormatter()
+        formatter.maximumFractionDigits = 0
+        data.setValueFormatter(DefaultValueFormatter(formatter: formatter))
+        data.setValueFont(UIFont(name: "Muli-Regular", size: 12)!)
+        data.setValueTextColor(UIColor(.aircastingGray))
```

Comment:
No need for double brackets, it can look like this: 
` data.setValueTextColor(UIColor.aircastingGray)`